aws/resource_aws_db_instance_test.go

```diff
@@ -3117,6 +3117,165 @@ func TestAccAWSDBInstance_RestoreToPointInTime_SourceResourceID(t *testing.T) {
 	})
 }
 
+func TestAccAWSDBInstance_CoipEnabled(t *testing.T) {
+	var v rds.DBInstance
+	resourceName := "aws_db_instance.bar"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
```

Comment:
So that the test won't fail when there is no pre-configured Outpost.
```suggestion
		PreCheck:     func() { testAccPreCheck(t); testAccPreCheckAWSOutpostsOutposts(t) },
```

Reply:
Thanks @ewbankkit, I have committed your suggestions. 😄